src/clients/github-client.js

```diff
@@ -164,6 +164,7 @@ export async function createRepository(
   repositoryDescription,
   username,
   password,
+  token,
```

Comment:
Can we change this to pass in the githubCredential object? It would seem weird to have to pass an empty value to this.